### ISSUE TITLE
rootURL not included in asset references inside CSS when prepending

### ISSUE THREAD

**Issue by Soliah** [ember-cli/ember-cli#6345]
```
ember-cli: 2.8.0
http_parser: 2.7.0
node: 4.5.0
v8: 4.5.103.37
uv: 1.9.1
zlib: 1.2.8
ares: 1.10.1-DEV
icu: 56.1
modules: 46
openssl: 1.0.2h
os: darwin x64
2.15.9
```

--

With `rootURL` specified the path is correctly included in the asset paths in `index.html`. My expectation was that this would also be included in the production built CSS paths for assets within the CSS like in `url()`. The assets do get correctly prepended, but without the `rootURL`.

Example repo here: https://github.com/Soliah/ember-rooturl-issue

`index.html` correctly has the following asset references:

    <link rel="stylesheet" href="https://aaaaaaaaaa.cloudfront.net/prefix/assets/vendor-d41d8cd98f00b204e9800998ecf8427e.css">
    <link rel="stylesheet" href="https://aaaaaaaaaa.cloudfront.net/prefix/assets/foo-cc63ae1c8911c16fbad619786f68cc64.css">
    <script src="https://aaaaaaaaaa.cloudfront.net/prefix/assets/vendor-46fbb40aa81c7f0e0ea83a08490a4742.js"></script>
    <script src="https://aaaaaaaaaa.cloudfront.net/prefix/assets/foo-ae4593dae4d95113bb240e617e8e71ef.js"></script>

However the CSS has the following:

```
background-image:url(https://aaaaaaaaaa.cloudfront.net/assets/images/red-2115f51cf86d128413541c5085b7a53e.png);
```

Doing a `ember build -prod` and then inspecting the built CSS illustrates the issue.

**Comment by Soliah** [comment]
A work around is to hardcode the path:

```css
background-image: { url('/prefix/assets/images/red.png'); }
```

This works in both dev and prod. The problem is my use case the CSS is being included from an addon where I can't hard code the `rootURL` ahead of time.

**Comment by nightire** [comment]
Why not append prefix path on https://github.com/Soliah/ember-rooturl-issue/blob/master/ember-cli-build.js#L11

**Comment by Soliah** [comment]
@nightire Thanks. Initially I found that didn't work but with @rwjblue suggestions in #6212 I was able to make it work.

So to summarise:

- Add rootURL prefix to the fingerprint statement: `https://aaaaaaa.cloudfront.net/prefix/`.
- In development leave `ENV.rootURL = '/prefix/';`
- In production set `ENV.rootURL = '/';`
- In `router.js` set `rootURL: '/prefix/'

This correctly sets the prefix in `index.html` and correctly rewrites asset references inside compiled CSS.
